include/aspect/simulator/assemblers/interface.h

```diff
@@ -107,6 +107,13 @@ namespace aspect
 
           FEValues<dim> finite_element_values;
 
+          void reinit ( const typename DoFHandler<dim>::active_cell_iterator &cell_ref)
```

Comment:
Our usual style would not have a space after the opening parenthesis.

Also, we generally declare the function in the class declaration, but the body of the function (the *definition*) goes into the .cc file. See where the body of the constructor is, for example.